data/human/free worlds reconciliation.txt

```diff
@@ -2540,6 +2540,8 @@ mission "FW Syndicate Extremists 1C"
 		dialog `You land on <planet>, but realize that your escort carrying Freya hasn't entered the system yet. Better depart and wait for it to arrive.`
 	on complete
 		log "Received an official commendation from Parliament and a promise that the war with the Free Worlds is officially over. Humanity is at peace once more, but new challenges may await in the star systems out beyond human space that are accessible only by jump drive."
+		log "Factions" "Free Worlds" `Tensions between the Free Worlds and the Republic mounted when additional systems decided to leave the Republic and join the Free Worlds, culminating in a series of military confrontations, that ended when the Free Worlds were able to present credible evidence denying any involvement in the terrorist attacks. After the Free Worlds joined the effort to drive out the Pug invaders, the Republic officially recognized the Free Worlds and diplomatic relations were established.`
+		log "Factions" "Syndicate" `The reputation of the Syndicate took further damage when it became apparent that a group of top management executives had conspired with the Alphas in an attempt to create a war between the Republic and the Free Worlds.`
```

Comment:
The way this log is worded sounds to me like it's coming after something else; "took further damage" implies to me that there's a prior sentence talking about an initial point in which their reputation was damaged, but this log doesn't exactly flow to me with the logs from the beginning of the game. (Even if it did flow, this wording then wouldn't make sense as soon as any other logs were added prior to this one.)

> in an attempt

They succeeded though, so I wouldn't simply call it an attempt.

How about this?
```suggestion
		log "Factions" "Syndicate" `It was exposed that the bombings of Gemini and Martini that helped cause the war between the Republic and Free Worlds were the doing of the Syndicate, designed by top executives with the help of a group of Alphas. Despite the claims that all the extremists responsible have been purged and handed over to the Navy, the reputation of the Syndicate will be damaged for years to come.`
```

Reply:
Your version is better, so I committed it, however, the current entry ends with: Their treatment of workers is questionable, and their opposition to organized labor is legendary. Some Syndicate factory towns have even been accused of human rights abuses.
That's the damaged reputation I was referring to.